ee/desktop/user/client/client.go

```diff
@@ -55,6 +59,41 @@ func (c *client) Refresh() error {
 	return c.get("refresh")
 }
 
+func (c *client) ShowDesktop() error {
+	return c.get("show")
+}
+
+func (c *client) DetectPresence(reason string, interval time.Duration) (time.Duration, error) {
+	encodedReason := url.QueryEscape(reason)
+	encodedInterval := url.QueryEscape(interval.String())
+
+	resp, requestErr := c.base.Get(fmt.Sprintf("http://unix/detect_presence?reason=%s&interval=%s", encodedReason, encodedInterval))
```

Comment:
Do we need a timeout here, or is there already one on c.base?

Reply:
yea, there is a default timeout of 30s on c.base

Reply:
Can you add a comment to that effect?

Reply:
done